.github/workflows/image-tests.yml

```
@@ -0,0 +1,144 @@
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    outputs:
+      image_tag: ${{ steps.set-tag.outputs.IMAGE_TAG }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v7.0.1
+
+      - name: Set image tag
+        id: set-tag
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:11:60: Double quote to prevent globbing and word splitting

(shellcheck)